emailer.py

```diff
@@ -199,7 +199,7 @@ def _get_subject(repo, message):
     else:
         subject_msg = message_lines[0]
     subject_msg = subject_msg[:50]
-    subject = '[{0}] {1}'.format(repo, subject_msg)
+    subject = '[Chapel Merge] {1}'.format(repo, subject_msg)
```

Comment:
```suggestion
    subject = '[Chapel Merge] {1}'.format(subject_msg)
```

Reply:
If we know this will always be using Python 3.6, we can use f-strings:

```suggestion
    subject = f'[Chapel Merge] {subject_msg}'
```

Reply:
I don't feel confident of that, and the last time I tried to change one of these scripts to assume any specific version of Python, I broke the world, so am somewhat gun-shy of those kinds of changes now.